api/queries/live-time-range-1d-developer/template.sql

```diff
@@ -0,0 +1,23 @@
+WITH pr_24h AS (
```

Comment:
You can refer the suggestion of live-time-range-1d-repo-pr.

the query name cloud be `live-time-top-repos-by-prs-daily` / `live-time-top-developers-by-prs-daily`..

Reply:
Done